cl/assets/react/disclosure-page.css

```diff
@@ -0,0 +1,84 @@
+/*Disclosure Table Viewer CSS*/
+
+.fa-exclamation-triangle {
+  font-size: 100px;
+}
+
+.search-panel-row {
+  border-bottom: 0.5px #cecece solid;
+  padding: 10px;
+}
+
+.search-panel-table {
+  width: 89%;
+  border-left: 1px solid;
+  border-right: 1px solid;
+  position: absolute;
+  z-index: 9;
+  background: white;
+  border-color: gainsboro;
+  border: 1px solid;
+  padding: 10px;
+  border-color: gainsboro;
+  box-shadow: 2px 2px 5px #acacac;
+}
+
+/*Disclosure List Homepage CSS*/
+.judge-pic {
+  margin-top: 10px;
+  min-width: 64px;
+  min-height: 64px;
+}
+
+.tr-results {
+  padding: 10px;
+  width: 100%;
+  background: white;
+  border-bottom: 2px solid gainsboro;
+}
+
+.table-instant-results {
+  width: 95%;
+  position: absolute;
+  z-index: 9;
+  box-shadow: 2px 2px 5px #acacac;
+}
+
+tr:hover {
+  background-color: whitesmoke;
+  text-decoration: none;
+}
+
+tr > a:hover {
+  text-decoration: none;
+  color: black;
+}
+
+.table-row-link {
+  text-decoration: none;
+  color: black;
+}
+
+.table-data-name {
+  width: 100%
+}
+
+.missing-judge {
+  font-size: 40px;
+  border: solid;
+  border-width: 1px;
+  margin: 0px;
+  padding: 12px;
+}
+
+.judge-name {
+  font-size: 20px;
+}
+
+.judge-court {
+  color: gray;
+}
+
+.hide-table {
+  display: None;
+}
```

Comment:
Doesn't bootstrap have a utility for this?

Reply:
yes - and on second review - we were doing it in one page but not the other.  its been fixed

Reply:
I think it's the same CSS though, isn't it? https://getbootstrap.com/docs/3.4/css/#helper-classes-show-hide